hanko-js/README.md

```diff
@@ -0,0 +1,81 @@
+# &lt;hanko-auth&gt; element
+
+The `<hanko-auth>` element offers a complete user interface that will bring a modern
+login and registration experience to your users. It integrates the Hanko API,
+a backend that provides the underlying functionalities.
+
+## Features
+
+* Registration and login flows with and without passwords
+* Platform Authenticators (e.g. Apple's Passkeys, Windows Hello, etc.)
```

Comment:
I'd prefer "Passkey authentication (on supported platforms)" instead of "Platform Authenticators (e.g. Apple's Passkeys, Windows Hello, etc.)" - Also: let's not say "Apple's passkeys" as it is a general term all platforms are using.

Reply:
I changed it, but left out "on supported platforms" because it's clear that it won't work then.